roommanager.js

```diff
@@ -225,6 +225,10 @@ class Room {
 				this.log.error(`Failed to get video info: ${err}`);
 				queueItem.title = queueItem.id;
 			}).then(() => {
+				if (this.queue.id.includes(queueItem.id) && this.queue.service.includes(queueItem.service)) {
```

Comment:
The condition should look more like: (https://lodash.com/docs/4.17.15#find)
```suggestion
				if (_.find(this.queue, queueItem)) {
```

And this check should go _before_ `getVideoInfo`